packages/block-editor/src/utils/index.js

```diff
@@ -1,3 +1,9 @@
 export { default as transformStyles } from './transform-styles';
 export * from './block-variation-transforms';
 export { default as getPxFromCssUnit } from './parse-css-unit-to-px';
+export {
+	PRESET_METADATA,
+	findInPresetsBy,
+	getValueFromVariable,
+} from './style-variable-resolution';
+export { getTypographyFontSizeValue } from './typography';
```

Comment:
I think these functions are too low level to export as `@wordpress/block-editor` APIs. In my mind they're implementation details of [`useGlobalStylesOuptut`](https://github.com/WordPress/gutenberg/blob/297565c13e84b07d17b9d08b1066927190a9f2bd/packages/edit-site/src/components/global-styles/use-global-styles-output.js#L886) and [`useStyle`](https://github.com/WordPress/gutenberg/blob/297565c13e84b07d17b9d08b1066927190a9f2bd/packages/edit-site/src/components/global-styles/hooks.js#L106). If we export them then it makes it difficult in the future to change the implementation of `useGlobalStylesOuptut` and `useStyle` without breaking backwards compatibility.